src/components/message/styles.scss

```diff
@@ -108,7 +108,7 @@
 
   &__block-audio {
     audio {
-      width: 100%;
+      width: 300px;
```

Comment:
Can you send an audio message _with_ a text message? If so, what should it look like? Maybe this might be more appropriate in case it's rendered with a long text message that expands the message box really wide. Perhaps, for example, a reply to a long message? I'm not sure if we have these scenarios or not.

```
width: 100%;
min-width: 300px;
```

Reply:
Pushed that change.

Before
![image](https://github.com/zer0-os/zOS/assets/43770/188ba1a1-dae8-48df-9935-6415a3d8b56f)

After
![image](https://github.com/zer0-os/zOS/assets/43770/25e15b14-c7bd-403c-a70c-aa218273205a)